pkg/operator/ceph/csi/spec.go

```diff
@@ -407,7 +407,9 @@ func StartCSIDrivers(namespace string, clientset kubernetes.Interface, ver *vers
 		// apply resource request and limit to rbd provisioner containers
 		applyResourcesToContainers(clientset, rbdProvisionerResource, &rbdProvisionerSTS.Spec.Template.Spec)
 		k8sutil.SetOwnerRef(&rbdProvisionerSTS.ObjectMeta, ownerRef)
-		err = k8sutil.CreateStatefulSet("csi-rbdplugin-provisioner", namespace, clientset, rbdProvisionerSTS)
```

Comment:
When is the statefulset ever created? I've never seen it created instead of the deployment.

Reply:
If the kubernetes version is 1.13.x the provisioner will be deployed as statefulset

Reply:
If we are planning to remove support for kube 1.13.x we can remove the same. Lot of features are not supported in kube 1.13 example resize snapshot clone etc

Reply:
1.13 support isn't being removed from Rook yet, sounds good to keep it if needed, I just didn't remember when the statefulset was used. Would it really not work as a deployment in 1.13? Anyway, this question is really independent from this PR.

Reply:
we could have used deployment with replica:1 I need to check why we used statefulset, not deployment?
one thing I can remember is the suggestion or example from the sidecar to use statefulset and later it moved to deployment

@ShyamsundarR any idea why it was statefulset instead of deployment